traffic_control/tests/test_decorators.py

```diff
@@ -18,19 +19,22 @@ def setUp(self):
     @override_settings(RATELIMIT_ENABLE=False)
     @override_settings(RATELIMIT_RATE="0/s")
     def test_valid_request_if_ratelimit_is_not_enabled(self):
+        assert getattr(self.rate_limit_view, RATELIMITED_VIEW_ATTR, None) is True
```

Comment:
Não seria interessante ter um teste que testa quando é `None` também?

Reply:
Não sei entendir... Não me parece ter  como testar esse cenário porque o atributo é setado pelo próprio decorator `enable_ratelimit`. Ele só é `None` se a view não for decorada com ele. Então, testar o `None` é testar o não-uso do código.

Reply:
Era mais para ter certeza de que o decorator/middleware não interfere quando não chamamos, mas concordo contigo.